### PR TITLE
When user removes automerge but leaves front_of_queue_automerge, cancel automerge

### PR DESCRIPTION
**Pull request by Strilanc**

### CODE REVIEW

File: dev_tools/auto_merge.py [hunk 1]
```diff
@@ -18,8 +18,9 @@
 ACCESS_TOKEN_ENV_VARIABLE = 'CIRQ_BOT_GITHUB_ACCESS_TOKEN'
 
 POLLING_PERIOD = datetime.timedelta(seconds=10)
+USER_AUTO_MERGE_LABEL = 'front_of_queue_automerge'
```
Comment (cduck) [inline dev_tools/auto_merge.py]: Did you mean `'automerge'`?

Comment (Strilanc) [reply dev_tools/auto_merge.py]: ...yes

Comment (Strilanc) [reply dev_tools/auto_merge.py]: Fixed

**Comment by Strilanc** [comment]
I was running the old version initially. Then it takes it ~45 seconds before it becomes confident that it's not a temporary thing.

**Pull request merged**